src/atomate2/common/schemas/cclib.py

```diff
@@ -297,7 +297,7 @@ def cclib_calculate(
 
     if method in cube_methods and not cube_file:
         raise FileNotFoundError(
-            f"A cube file must be provied for {method}. Returning None."
+            f"A cube file must be provide for {method}. Returning None."
```

Comment:
I think it should actually be "provided"

Reply:
Good catch.